app/shared/text-editor/config.ts

```diff
@@ -0,0 +1,11 @@
+import { InstanceNode } from "./nodes/node-instance";
+import { darkTheme } from "./themes/dark";
+
+export const config = {
+  namespace: "example",
+  theme: darkTheme,
```

Comment:
Not sure it should be hard-coded here like that but rather depends on the user preference. Can this be abstracted away?

Reply:
need to figure this out yet, theme is basically just a map to class names

Reply:
removed